### PR TITLE
Rfc0044 rename

### PR DESCRIPTION
**Pull request by dirvine**
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/rfcs/191)

### CODE REVIEW

File: text/0044-relay-nodes/0044-relay-nodes.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Secure Vault Joining
+# Relaynodes
```
Comment (frabrunelle) [inline text/0044-relay-nodes/0044-relay-nodes.md]: It could be written as "Relay Nodes" instead 😃

Comment (dirvine) [inline text/0044-relay-nodes/0044-relay-nodes.md]: Nice catch, done now.

File: text/0044-relay-nodes/0044-relay-nodes.md [hunk 2]
```diff
@@ -31,6 +31,8 @@ of such an attack exists in some form.
 This RFC allows the network to ensure that joining nodes are tested prior to joining a group. As a
 node is promoted then it can begin to acquire rank. This RFC does not cover rank or relocation.
 
+Another important feature of This RFC is that the
```
Comment (frabrunelle) [inline text/0044-relay-nodes/0044-relay-nodes.md]: It seems like part of this sentence is missing 😛

Comment (dirvine) [inline text/0044-relay-nodes/0044-relay-nodes.md]: Ah thanks - removed this now.

**Comment by frabrunelle** [comment]
LGTM 👍

**Pull request merged**